modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,102 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that it's possible to send an e-mail notification via an Action (Delete)"
+	@priority = 5
+	test CanSendEmailViaActionDelete {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ObjectNotifications#CanSendEmailViaActionDelete";
+
+		task ("Given: an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And: an Custom Object with an Action (Delete Entry)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 178781",
+				objectName = "CustomObject178781",
+				pluralLabelName = "Custom Objects 178781");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject178781");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject178781");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 178781");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				notificationName = "Notification Template",
+				objectLabel = "Custom Object 178781",
+				thenAction = "Notification",
+				whenAction = "On After Delete");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject178781",
+				value = "Entry Test");
+		}
+
+		task ("When: the timing is set on Notification Configuration") {
```

Comment:
This is part of the given since it's part of the test setup. Don't leave the wording vague, you should specify what we're setting the timing to.
`And given the Notification Queue Entry Delete Interval is set to 1 in System Settings > Notifications > Notification Queue`